Bug on formatting codes SB 2.2.2

**Describe the bug**
SB not showing when formatting codes in placeholders uses § instead of &

**To Reproduce**
Steps to reproduce the behavior:
1. Just put a placeholder in the sb that returns § as color or code
2. Type /sb toggle and it hapens

**Expected behavior**
The board shows normally as it is a color

Console Log:

[17:30:02 INFO]: zSkyman issued server command: /sb reload
[17:30:02 INFO]: zSkyman issued server command: /sb toggle
[17:30:02 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'sb' in plugin SternalBoard v2.2.1
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[paper-api-1.20.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155) ~[paper-api-1.20.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_20_R1.CraftServer.dispatchCommand(CraftServer.java:1003) ~[paper-1.20.1.jar:git-Paper-48]
        at org.bukkit.craftbukkit.v1_20_R1.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:64) ~[paper-1.20.1.jar:git-Paper-48]
        at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:265) ~[paper-1.20.1.jar:?]
        at net.minecraft.commands.Commands.performCommand(Commands.java:324) ~[?:?]
        at net.minecraft.commands.Commands.performCommand(Commands.java:308) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.performChatCommand(ServerGamePacketListenerImpl.java:2299) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.lambda$handleChatCommand$20(ServerGamePacketListenerImpl.java:2259) ~[?:?]
        at net.minecraft.util.thread.BlockableEventLoop.lambda$submitAsync$0(BlockableEventLoop.java:59) ~[?:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.20.1.jar:git-Paper-48]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1339) ~[paper-1.20.1.jar:git-Paper-48]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197) ~[paper-1.20.1.jar:git-Paper-48]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1316) ~[paper-1.20.1.jar:git-Paper-48]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1309) ~[paper-1.20.1.jar:git-Paper-48]
        at net.minecraft.util.thread.BlockableEventLoop.runAllTasks(BlockableEventLoop.java:114) ~[?:?]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1425) ~[paper-1.20.1.jar:git-Paper-48]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1171) ~[paper-1.20.1.jar:git-Paper-48]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:318) ~[paper-1.20.1.jar:git-Paper-48]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: net.kyori.adventure.text.minimessage.internal.parser.ParsingExceptionImpl: Legacy formatting codes have been detected in a MiniMessage string - this is unsupported behaviour. Please refer to the Adventure documentation (https://docs.advntr.dev) for more information.
        &a1&b/&6§6Ilimitado
                ^^
        at net.kyori.adventure.text.minimessage.internal.parser.TokenParser.parseString(TokenParser.java:170) ~[adventure-text-minimessage-4.14.0.jar:?]
        at net.kyori.adventure.text.minimessage.internal.parser.TokenParser.resolvePreProcessTags(TokenParser.java:112) ~[adventure-text-minimessage-4.14.0.jar:?]
        at net.kyori.adventure.text.minimessage.MiniMessageParser.parseToTree(MiniMessageParser.java:195) ~[adventure-text-minimessage-4.14.0.jar:?]
        at net.kyori.adventure.text.minimessage.MiniMessageParser.parseFormat(MiniMessageParser.java:209) ~[adventure-text-minimessage-4.14.0.jar:?]
        at net.kyori.adventure.text.minimessage.MiniMessageImpl.deserialize(MiniMessageImpl.java:79) ~[adventure-text-minimessage-4.14.0.jar:?]
        at net.kyori.adventure.text.minimessage.MiniMessageImpl.deserialize(MiniMessageImpl.java:45) ~[adventure-text-minimessage-4.14.0.jar:?]
        at com.xism4.sternalboard.utils.TextUtils.colorize(TextUtils.java:39) ~[SternalBoard-2.2.2-all.jar:?]
        at com.xism4.sternalboard.utils.TextUtils.processPlaceholders(TextUtils.java:50) ~[SternalBoard-2.2.2-all.jar:?]
        at com.xism4.sternalboard.Scoreboards.lambda$updateFromSection$0(Scoreboards.java:28) ~[SternalBoard-2.2.2-all.jar:?]
        at java.util.ArrayList.replaceAllRange(ArrayList.java:1712) ~[?:?]
        at java.util.ArrayList.replaceAll(ArrayList.java:1702) ~[?:?]
        at com.xism4.sternalboard.Scoreboards.updateFromSection(Scoreboards.java:28) ~[SternalBoard-2.2.2-all.jar:?]
        at com.xism4.sternalboard.managers.ScoreboardManager.setScoreboard(ScoreboardManager.java:81) ~[SternalBoard-2.2.2-all.jar:?]
        at com.xism4.sternalboard.managers.ScoreboardManager.toggle(ScoreboardManager.java:111) ~[SternalBoard-2.2.2-all.jar:?]
        at com.xism4.sternalboard.commands.SternalCommand.toggleSubcommand(SternalCommand.java:81) ~[SternalBoard-2.2.2-all.jar:?]
        at com.xism4.sternalboard.commands.SternalCommand.onCommand(SternalCommand.java:42) ~[SternalBoard-2.2.2-all.jar:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper-api-1.20.1-R0.1-SNAPSHOT.jar:?]
        ... 23 more

> Caused by: net.kyori.adventure.text.minimessage.internal.parser.ParsingExceptionImpl: Legacy formatting codes have been detected in a MiniMessage string

Caused by: net.kyori.adventure.text.minimessage.internal.parser.ParsingExceptionImpl: Legacy formatting codes have been detected in a MiniMessage string

--- a/plugin/build.gradle.kts
+++ b/plugin/build.gradle.kts
@@ -38,3 +38,9 @@ tasks {
         delete("${rootDir}/bin/")
     }
 }
+
+tasks.processResources {
+    filesMatching("plugin.yml") {
+        expand("version" to (parent?.version ?: project.version))
+    }
+}

--- a/plugin/src/main/java/com/xism4/sternalboard/utils/TextUtils.java
+++ b/plugin/src/main/java/com/xism4/sternalboard/utils/TextUtils.java
@@ -32,6 +32,7 @@ public class TextUtils {
         }
 
         text = transformLegacyHex(text);
+        text = text.replaceAll("\u00a7", "&");
 
         MiniMessage mm = MiniMessage.miniMessage();
         text = LegacyComponentSerializer.legacySection().serialize(mm.deserialize(text));

--- a/plugin/src/main/resources/plugin.yml
+++ b/plugin/src/main/resources/plugin.yml
@@ -1,7 +1,7 @@
 name: SternalBoard
 main: com.xism4.sternalboard.SternalBoardPlugin
 authors: [ xIsm4, Jonakls ]
-version: 2.2.2
+version: '${version}'
 api-version: "1.13"
 softdepend: [ PlaceholderAPI ]
 commands:

